src/Lifeforms.sol

```diff
@@ -54,7 +54,10 @@ contract Lifeforms is ERC721, NFTSVG {
     uint256 public immutable BASE_UNIT;
 
     /// @notice Maximum number of token instances that can be minted on this contract.
-    uint256 public maxSupply;
+    uint256 public immutable maxSupply;
+
+    /// @notice Deployer of the contract that can withdraw ETH
+    address public deployer;
```

Comment:
In a previous iteration I had used an Ownable for claiming the ETH, I think I'll add that back in favour of adding a deployer.

Reply:
I wanted to do that but i said that must be discussed with you! 😬

Reply:
I can do it if you want or you can merge it and do another commit